src/sentry/sentry_apps/api/endpoints/installation_external_issue_actions.py

```diff
@@ -71,8 +74,16 @@ def post(self, request: Request, installation) -> Response:
                 uri=uri,
                 user=user,
             ).run()
-        except Exception:
-            return Response({"error": "Error communicating with Sentry App service"}, status=400)
+        except (APIError, ValidationError, APIUnauthorized) as e:
```

Comment:
Where are these 3 error types being raised from? I'd err on the side of caution by not providing the explicit exception via the API unless we can ensure it's not going to leak anything of consequence.

Reply:
The callsites for each of those errors is here:
- [APIUnauthorized](https://github.com/getsentry/sentry/blob/master/src/sentry/sentry_apps/external_issues/issue_link_creator.py#L33-L35)
- [ValidationError](https://github.com/getsentry/sentry/pull/80033/files#diff-01f93072bc4ab2c5c6468c2c5afd04b811388c74c118cc7234b603c8a08b05bbR92-R94)
- [ApiUnauthorized](https://github.com/getsentry/sentry/pull/80033/files#diff-01f93072bc4ab2c5c6468c2c5afd04b811388c74c118cc7234b603c8a08b05bbR88-R90)

Yeah I definitely understand, the biggest thing that would leak would be the stacktrace imo. Additionally, the errors we are using are pretty specific and unlikely to be thrown by any of the supporting functions.

Alternatively, if we want to be super safe, we could do a switch case for each of the error types and print a generic message. It just kinda sucks since, we already wrote the messages earlier.

Reply:
`Error communicating with Sentry App service` is really terrible because some of these errors have nothing to do with sentry at all. People read it as trouble communicating with Sentry. So if we want to use generic error but more informative one I would be ok with it.